internal/server/v1/propertyvalues/golden/property_values_in/typeOf1.json

```diff
@@ -87,7 +87,7 @@
         "Country"
       ],
       "dcid": "country/ARM",
-      "provenanceId": "dc/sgvysq3"
+      "provenanceId": "dc/5n63hr1"
```

Comment:
provenance diffs are an intended improvement that hopefully will remain stable henceforth (cl/485779075)